packages/cactus-plugin-satp-hermes/src/test/typescript/unit/services.test.ts

```diff
@@ -170,6 +171,8 @@ describe("SATP Services Testing", () => {
     sessionData.credentialProfile = CredentialProfile.X509;
     sessionData.loggingProfile = "MOCK_LOGGING_PROFILE";
     sessionData.accessControlProfile = "MOCK_ACCESS_CONTROL_PROFILE";
+    sessionData.resourceUrl = "MOCK_RESOURCE_URL";
+    sessionData.lockAssertionExpiration = BigInt(99999);
```

Comment:
likewise for expiry

Reply:
This is not applicable here, in the test file